### DISCUSSION TITLE
Support WaveDrom in README markdown page

### DISCUSSION THREAD

**Discussion by ShortArrow** [community/community#140157] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Mermaid is supported [now](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams).

So will GitHub have plan to support [WaveDrom](https://github.com/wavedrom/wavedrom) ?

I think it would be very useful for an embedded electronics repository, or even a physical layer repository.

**Comment by xiaoweish** [comment]
Yes, RISC-V and many other electronics repositories use WaveDrom to create diagrams. It is an useful tool for electronics engineers. For example, see:
https://github.com/riscv/riscv-isa-manual/blob/main/src/machine.adoc
